### PR TITLE
Add Validation to form

### PR DESCRIPTION
**Pull request by mrparkersson**
Pull request
Amo Parker Prince

What does this PR do?

Add validation to email input to check for all inputs whether it matches an email

Description of the task to be completed
Adding clicks in Javascript file

Add validation to email input to check for all inputs whether it matches an email

clone the repository
git clone https://github.com/mrparkersson/mobile-version.git
navigate to the folder
cd portfolio-skeleton
checkout the feature
javascript-create-my-page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -206,44 +189,111 @@ <h2 class="about">About Me</h2>
...
               required
             />
+
             <input
+              id="email"
               type="email"
               name="email"
               placeholder="Email address"
               required
             />
```
Comment (oloomoses) [inline index.html:214]: - [ ] email field containing uppercase characters should generate an error as per the project requirements. You can check if the email is all lowercase by comparing if the email and it's lowercase converted version are the same, ie: `email === email.toLowerCase()`, utilizing regular expressions, or looping through each letter of the email string.

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hi team 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mrparkersson) [reply index.html:214]: if it optional why didn't you just approve for me to do. than to give me changes required in my dashboard?

Comment (mrparkersson) [reply index.html:214]: Besides it is the same logic. just read please

**Review by Rustamxon7 — APPROVED** [review]
## Hi @mrparkersson   👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

batman aproves

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**